docs/developer/activity-embeds.md

```diff
@@ -0,0 +1,174 @@
+# BuddyPress Embeds Documentation
+
+## Overview
+
+BuddyPress introduced support for embedding single activity items starting from version 2.6.0, utilizing the WordPress oEmbed functionality. This allows activity items to be embedded within WordPress posts or pages by pasting the activity permalink into the editor.
+
+### Embedding Activity Items
+
+Users can embed a BuddyPress activity item by copying the activity permalink and pasting it into a WordPress post or page. The embedded activity will display content, media (if applicable), and comments.
+
+---
+
+## Disabling Activity Embeds
+
+If you prefer not to use activity embeds on your BuddyPress site, you can disable the feature using the following code snippet:
+
+```php
+add_filter( 'bp_is_activity_embeds_active', '__return_false' );
+```
```

Comment:
This part should probably be in the hooks page @emaralive is thinking about, see https://github.com/buddypress/bp-documentation/discussions/292